### DISCUSSION TITLE
更新后关系图谱不显示

### DISCUSSION THREAD

**Discussion by Wenwen-ai-cell** [MuiseDestiny/zotero-style#265] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/78087414/225792544-61c1c61e-a531-4862-967a-81f616dd300f.png)
这是我更新了最新版之后的显示，关系图谱没有了，不知道是什么情况呢

**Comment by Wenwen-ai-cell** [comment]
PS：已经选中论文，且论文由关联文献

**Comment by MuiseDestiny** [comment, marked as answer]
请鼠标滑动到关系图谱最底部，点击几个红点切换。如果切换成功请关闭此讨论。

**Comment by Wenwen-ai-cell** [comment]
答主给的意见很正确，切换按钮就可以解决，非常感谢！！
